src/test/java/org/broadinstitute/hellbender/tools/copynumber/CollectFragmentCountsIntegrationTest.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.test.IntegerReadCountFileComparator;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * Integration test for {@link CollectFragmentCounts}.
+ *
+ * @author Andrey Smirnov &lt;asmirnov@broadinstitute.org&gt;
+ */
+public class CollectFragmentCountsIntegrationTest extends CommandLineProgramTest {
+
+    private static final String TEST_SUB_DIR = publicTestDir + "org/broadinstitute/hellbender/tools/copynumber/collectfragmentcounts";
+
+    private static final File NA12878_BAM = new File(TEST_SUB_DIR, "exome-read-counts-NA12878.bam");
```

Comment:
Let's also be more consistent in naming our test files.  All test files should start with the name of the tool/class, e.g., "collect-fragment-counts-*".  So perhaps something along the lines of "collect-fragment-counts-NA12878-exome.bam".